### ISSUE TITLE
Show values in Time Series panel

### ISSUE THREAD

**Issue by SC-66** [grafana/grafana#77687]
**Why is this needed**:
I need to use **lines** to display several series in the same chart. I can therefore not use bar chart to show the value.
I know bar charts can show the value on top of the bar, but the data I need to show is required to show as lines.

**What would you like to be added**:
Add an option to show value of the point. 

**Who is this feature for?**
All users. This is a basic feature for a time line panel.

![image](https://github.com/grafana/grafana/assets/92076711/c4a52d9b-5ec1-44f1-82e1-5e5fcc18fcf9)

**Comment by szechp** [comment]
this is an essential feature, a bar chart without time series already has this.

**Comment by mirecta** [comment]
Yes Bar chart has it but has not pan and zoom and Time series panel has

**Comment by leeoniya** [comment]
https://github.com/grafana/grafana/pull/108090 adds this :)
